src/main/resources/templates/templateOpportunityNotification.xml

```diff
@@ -1,56 +1,63 @@
             <header>
-                <img src="${entity['OpportunityNotification'].current.opportunity.IconCrown}" class="icon" />
-                <span class="tempo-text-color--secondary">Opportunity: </span>
-                <a class="tempo-text-color--link" href="${entity['OpportunityNotification'].current.opportunity.Link}">${entity['OpportunityNotification'].current.opportunity.Name}</a>
-                <span class="tempo-text-color--green"> - ${entity['OpportunityNotification'].current.opportunity.CreatedOrUpdated}</span>
-                <span>${entity['OpportunityNotification'].current.opportunity.UpdatedFields}</span>
+                <img src="${entity['OpportunityNotification'].current.opportunity.IconCrown}" class="icon"/>
+                <span class="tempo-text-color--normal">Opportunity:</span>
+                <#if (entity['OpportunityNotification'].current.opportunity.Link)??>
+                    <a class="tempo-text-color--link" href="${entity['OpportunityNotification'].current.opportunity.Link}">${entity['OpportunityNotification'].current.opportunity.Name ! "-"}</a>
+                <#else>
+                    <span>${entity['OpportunityNotification'].current.opportunity.Name ! "-"}</span>
+                </#if>
+                <span class="tempo-text-color--green">- ${entity['OpportunityNotification'].current.opportunity.CreatedOrUpdated}</span>
+                <#if (entity['OpportunityNotification'].current.opportunity.UpdatedFields)??>
+                    <span>${entity['OpportunityNotification'].current.opportunity.UpdatedFields}</span>
+                </#if>
             </header>
             <body>
                 <div>
                     <span class="tempo-text-color--secondary">Account Name:</span>
                     <#if (entity['OpportunityNotification'].current.opportunity.Account.Link)??>
-                        <a class="tempo-text-color--link" href="${entity['OpportunityNotification'].current.opportunity.Account.Link}">${entity['OpportunityNotification'].current.opportunity.Account.Name}</a>
+                        <a class="tempo-text-color--link" href="${entity['OpportunityNotification'].current.opportunity.Account.Link}">${entity['OpportunityNotification'].current.opportunity.Account.Name ! "-"}</a>
                     <#else>
                         <span>${entity['OpportunityNotification'].current.opportunity.Account.Name}</span>
                     </#if>
                 </div>
                 <br/>
                 <div>
                     <span class="tempo-text-color--secondary">Opportunity Owner:</span>
-                    <#if (entity['OpportunityNotification'].current.opportunity.Owner.hasOwnerAtSymphony)??>
-                        <mention email="${entity['OpportunityNotification'].current.opportunity.Owner.Email}" />
+                    <#if (entity['OpportunityNotification'].current.opportunity.Owner.hasOwnerAtSymphony == "true")>
```

Comment:
I think you don't need to put == "true" here, if the JSON attribute is a boolean field (as it should be).